src/mame/moog/source.cpp

```diff
@@ -408,7 +408,7 @@ float source_state::get_keyboard_v() const
 
 	// *** Convert pressed key to a voltage.
 
-	static constexpr const float KEYBOARD_VREF = 8.24;  // From schematic.
+	static constexpr const float KEYBOARD_VREF = 8.24f;  // From schematic.
```

Comment:
Does MSVC really require literal suffixes for constant initialisation of `float` variables?  That doesn’t strike me as useful.

Reply:
It treats it as a cast from double to float, (loss of precision).